### PR TITLE
[myq] Fixes breaking API changes to the MyQ binding

### PR DESCRIPTION
**Pull request by digitaldan**
Signed-off-by: Dan Cunningham <dan@digitaldan.com>

### CODE REVIEW

**Comment by wborn** [comment]
The PR builds are failing because this add-on bundle is missing from the add-ons BOM, which should be fixed with https://github.com/openhab/openhab-addons/pull/11605.

**Comment by digitaldan** [comment]
thanks @wborn !  Did i forget to add that when this binding was first submitted?  Yikes. 😱

File: bundles/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/handler/MyQAccountHandler.java [hunk 1]
```diff
@@ -182,33 +188,42 @@ public void onAccessTokenResponse(AccessTokenResponse tokenResponse) {
     }
 
     /**
-     * Sends an action to the MyQ API
+     * Sends a door action to the MyQ API
+     *
+     * @param devivce
```
Comment (wborn) [inline bundles/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/handler/MyQAccountHandler.java:193]:
```suggestion
     * @param device
```

File: bundles/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/handler/MyQAccountHandler.java [hunk 2]
```diff
@@ -182,33 +188,42 @@ public void onAccessTokenResponse(AccessTokenResponse tokenResponse) {
...
+     * @param action
+     */
+    public void sendDoorAction(DeviceDTO devivce, String action) {
```
Comment (wborn) [inline bundles/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/handler/MyQAccountHandler.java:196]:
```suggestion
    public void sendDoorAction(DeviceDTO device, String action) {
```

File: bundles/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/handler/MyQAccountHandler.java [hunk 3]
```diff
@@ -182,33 +188,42 @@ public void onAccessTokenResponse(AccessTokenResponse tokenResponse) {
...
+        sendAction(devivce, action, CMD_DOOR_URL);
+    }
+
+    /**
+     * Sends a lamp action to the MyQ API
      *
-     * @param serialNumber
+     * @param devivce
```
Comment (wborn) [inline bundles/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/handler/MyQAccountHandler.java:203]:
```suggestion
     * @param device
```

File: bundles/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/handler/MyQAccountHandler.java [hunk 4]
```diff
@@ -182,33 +188,42 @@ public void onAccessTokenResponse(AccessTokenResponse tokenResponse) {
...
      * @param action
      */
-    public void sendAction(String serialNumber, String action) {
+    public void sendLampAction(DeviceDTO devivce, String action) {
```
Comment (wborn) [inline bundles/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/handler/MyQAccountHandler.java:206]:
```suggestion
    public void sendLampAction(DeviceDTO device, String action) {
```

File: bundles/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/handler/MyQAccountHandler.java [hunk 5]
```diff
@@ -182,33 +188,42 @@ public void onAccessTokenResponse(AccessTokenResponse tokenResponse) {
...
+        sendAction(devivce, action, CMD_LAMP_URL);
```
Comment (wborn) [inline bundles/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/handler/MyQAccountHandler.java:207]:
```suggestion
        sendAction(device, action, CMD_LAMP_URL);
```

**Review by wborn — CHANGES_REQUESTED** [review]
Thanks for the fixes @digitaldan! The code looks good but I think there are a couple of typos.

**Comment by digitaldan** [comment]
Thanks @wborn , i really copied and pasted the heck out of that typo !   That should be fixed now.

**Review by wborn — APPROVED** [review]

**Pull request merged**